PurchasesCoreSwift/Logging/Strings/AttributionStrings.swift

```diff
@@ -13,17 +13,17 @@ import Foundation
     @objc public var appsflyer_id_deprecated: String {
         "The parameter key rc_appsflyer_id is deprecated. Pass networkUserId to addAttribution instead."
     }
-    @objc public var attributes_sync_error: String {
+    public var attributes_sync_error: String {
```

Comment:
I just happened to try this in another PR. Since this is still referenced in `Strings.swift`, and that's `public` we can't switch them yet to `internal` 😢  maybe Monday 😄